### PR TITLE
docs: Refactored the quickstart to capture our latest understanding 

### PR DESCRIPTION
**Pull request by mieubrisse**
## Description:
This change is to refactor the quickstart using our latest understanding of what value users are derived out of Kurtosis. It also attempts to standardize the CLI installation/upgrading/historical-installation docs, and adds tabs to boot.

## Is this change user facing?

* [x] Yes
* [ ] No

Changelog picked up from commits here:

fix: address bugs derek ran into

### CODE REVIEW

File: docs/docs/quickstart.md [hunk 1]
```diff
@@ -669,13 +669,13 @@ You can paste the URL from your output into your browser (or Cmd+click it in iTe
 Now make a request to insert a row into the database (replacing the `http://127.0.0.1:59992` portion of the URL with the correct URL from your `enclave inspect` output)...
 
 ```bash
-curl -XPOST -H "content-type: application/json" http://127.0.0.1:59992/actor --data '{"first_name": "Kevin", "last_name": "Bacon"}'
+curl -XPOST -H "content-type: application/json" http://127.0.0.1:<YOUR_URL>/actor --data '{"first_name": "Kevin", "last_name": "Bacon"}'
```
Comment (mieubrisse) [inline docs/docs/quickstart.md]: 1. URL = `http://` prefix + IP + port + optional path; the current `<YOUR URL>` is actually just the port
2. This doesn't line up with the instructions above this command and åbove the one below to `replacing the http://127.0.0.1:59992 portion of the URL with the correct URL from your enclave inspect output`

File: docs/docs/quickstart.md [hunk 2]
```diff
@@ -800,7 +800,7 @@ kurtosis clean -a && kurtosis run --enclave-identifier quickstart . '[{"first_na
 Using the new `http` URL on the `postgrest` service in the output, query for the rows you just added (replacing `http://127.0.0.1:59992` with your URL)...
 
 ```bash
-curl -XGET "http://127.0.0.1:59992/actor?or=(last_name.eq.Buscemi,last_name.eq.Bacon)"
+curl -XGET "http://127.0.0.1:<YOUR_URL>/actor?or=(last_name.eq.Buscemi,last_name.eq.Bacon)"
```
Comment (mieubrisse) [inline docs/docs/quickstart.md]: same problem

File: docs/docs/quickstart.md [hunk 3]
```diff
@@ -863,7 +863,7 @@ git add . && git commit -m "Initial commit" && git push origin main
 Now that your package is live, any Kurtosis user can run it using their CLI without even cloning your repo:
 
 ```bash
-kurtosis clean -a && kurtosis run --enclave-identifier quickstart github.com/mieubrisse/kurtosis-quickstart
+kurtosis clean -a && kurtosis run --enclave-identifier quickstart github.com/YOUR-GITHUB-USERNAME/kurtosis-quickstart
```
Comment (mieubrisse) [inline docs/docs/quickstart.md:866]: thanks 💪

File: docs/docs/quickstart.md [hunk 4]
```diff
@@ -169,7 +169,7 @@ Both are possible in Kurtosis, but for this tutorial we'll do the second one usi
 
 Normally seeding a database would require downloading the seed data to your machine, starting Postgres, and writing a pile of Bash to copy the seed data to the Postgres server and run a `pg_restore`. If you forgot to check if the database is available, you may get flakes when you try to use the seeding logic in a test. 
 
-You could try Docker Compose to volume-moun the data TAR into the Postgres server, but you'd still need to handle Postgres availability and sequencing the `pg_restore` afterwards.
+You could try Docker Compose to volume-mount the data TAR into the Postgres server, but you'd still need to handle Postgres availability and sequencing the `pg_restore` afterwards.
```
Comment (mieubrisse) [inline docs/docs/quickstart.md:172]: oops

File: docs/docs/quickstart.md (hunk 1 above)
Comment (leeederek) [reply docs/docs/quickstart.md]: Ah got it. Thanks. Let's try:
```suggestion
curl -XPOST -H "content-type: application/json" <YOUR_HTTP_URL>/actor --data '{"first_name": "Kevin", "last_name": "Bacon"}'
```

File: docs/docs/quickstart.md (hunk 2 above)
Comment (leeederek) [reply docs/docs/quickstart.md]:
```suggestion
curl -XGET "<YOUR_HTTP_URL>/actor?or=(last_name.eq.Buscemi,last_name.eq.Bacon)"
```

**Review by leeederek — APPROVED** [review]
Looks amazing to me. Thanks for turning this around!

**Pull request merged**